app/src/main/res/layout/item_edit_history.xml

```diff
@@ -83,9 +85,10 @@
             app:iconPadding="8dp"
             app:iconSize="16dp"
             app:iconTint="?attr/material_theme_secondary_color"
+            app:layout_constraintBaseline_toBaselineOf="@+id/diffText"
             app:layout_constraintStart_toEndOf="@id/diffText"
-            app:layout_constraintTop_toTopOf="@id/diffText"
             app:layout_constraintWidth_max="220dp"
+            app:rippleColor="@null"
```

Comment:
Looks like you missed this one. We can have `MaterialTextView` for this one.

Reply:
We'll discuss this during sync, but I kind of disagree that this should no longer be a button, because (a) it looks like a button, and (b) it would be very useful to tap a username and provide options to go to the user's Talk page or Profile page, as we do in other places.